src/components/icon/icon-test.stories.tsx

```diff
@@ -112,6 +112,7 @@ export const All = () => (
               fontSize={fontSize}
               bgShape={bgShape}
               bgSize={bgSize}
+              bg="#00b000"
```

Comment:
**question**: just wondering if there's a reason why this background colour has been changed

Reply:
I assume it's so the background is actually visible, to better see the different sizes of background vs icon. (It took me a little while to realise that you have to scroll down a fair way to see the icons with this green background, as there are lot of icons before these ones in the story!)

Reply:
Robin is correct here - I noticed that the background was not visible even if it has been set so adding a background colour made changes easier to see.